x-pack/test/apm_api_integration/trial/tests/service_maps/__snapshots__/service_maps.snap

```diff
@@ -1,1995 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`Service Maps with a trial license /api/apm/service-map when there is data returns service map elements filtering by environment not defined 1`] = `
```

Comment:
@dgieselaar Does this look okay to you? (removing all the snapshots)

Reply:
Jest integration tests donot currently run for X-Pack - I have a draft PR up showing that removing these does not effect the current CI jobs. https://github.com/elastic/kibana/pull/84783